templates/cluster-tsp/input.json

```diff
@@ -0,0 +1,91 @@
+{
+  "vehicles": ["v1", "v2"],
+  "stops": [
```

Comment:
The names of the stops seem a little strange, e.g., `West` vs. `Place 1`. However, it's not a blocker.